lib/screens/calendar/calendar.dart

```diff
@@ -0,0 +1,841 @@
+                        Text(
+                          '${entriesList[i].title}',
+                          style: TextStyle(
+                              fontWeight: FontWeight.w700,
+                              fontSize: 17,
+                              fontFamily:
+                                  'assets\fonts\Barlow\Barlow-Regular.ttf',
+                              color:
+                                  isClassOfEpisodeCalendarEntry(entriesList[i])
+                                      ? Colors.white
+                                      : Colors.white54),
+                        ),
+                        SizedBox(height: 4),
+                        Text('${entriesList[i].description}',
+                            style: TextStyle(
+                              color: (DateTime.parse(entriesList[i].end)
+                                          .isAfter(DateTime.now()) &&
+                                      DateTime.parse(entriesList[i].end)
+                                          .isBefore(DateTime.now()))
+                                  ? isClassOfEpisodeCalendarEntry(
+                                          entriesList[i])
+                                      ? Colors.red
+                                      : Colors.pink[800]
+                                  : Color.fromRGBO(110, 176, 230, 1),
+                              fontWeight: FontWeight.w500,
+                              fontSize: 14,
+                              fontFamily:
+                                  'assets\fonts\Barlow\Barlow-Regular.ttf',
+                            )),
+                      ],
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          TvGuideTime()
+        ] else ...[
+          Padding(
+            padding: const EdgeInsets.symmetric(vertical: 36.0),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Text(
+                  'Ingen planlagte sendinger',
+                  style: TextStyle(
+                    color: Colors.white,
+                    fontWeight: FontWeight.w700,
+                    fontSize: 17,
+                    fontFamily: 'assets\fonts\Barlow\Barlow-Regular.ttf',
+                  ),
+                )
+              ],
+            ),
+          )
+        ]
+      ],
+    );
+  }
+}
+
+class TvGuideTime extends StatefulWidget {
+  const TvGuideTime();
+
+  @override
+  State<TvGuideTime> createState() => _TvGuideTimeState();
+}
+
+class _TvGuideTimeState extends State<TvGuideTime> {
+  String time =
+      DateTime.now().hour.toString() + ":" + DateTime.now().minute.toString();
+  late Timer mytimer;
+
+  @override
+  void initState() {
+    mytimer = Timer.periodic(Duration(seconds: 1), (timer) {
+      time = DateTime.now().hour.toString() +
+          ":" +
+          DateTime.now().minute.toString();
+      setState(() {});
```

Comment:
best practice is to put the time = inside the setState.
Like `setState(() { time = ... })`